trieste/models/normalization/model_wrappers.py

```diff
@@ -0,0 +1,232 @@
+from typing import Mapping, Any, Tuple, Sequence, Union
+import tensorflow as tf
+from gpflow import default_float
+from gpflow.utilities.traversal import read_values, multiple_assign
+from trieste.models.gpflow.interface import GPflowPredictor
+
+from trieste.models.gpflow.models import GaussianProcessRegression
+
+from .transformers import DataTransformer
+from ..interfaces import TrainableProbabilisticModel
+from ...data import Dataset
+from ...types import S, TensorType
+
+
+class DataTransformWrapper(TrainableProbabilisticModel):
```

Comment:
better but not there yet :)
I think its better to have a word model in there as well, so that user can immediately see that this is something that is used on a model